include/deal.II/base/thread_local_storage.h

```diff
@@ -256,8 +274,6 @@ namespace Threads
      * An exemplar for creating a new (thread specific) copy.
      */
     std::shared_ptr<const T> exemplar;
-
-    friend class dealii::LogStream;
```

Comment:
Unfriending someone on social media is often considered to be mean. Here, I think, it is OK.